webapp/src/components/Staking/StakingPools.tsx

```diff
@@ -370,8 +387,18 @@ const StakingPool: React.FC<StakingPoolProps> = ({ vaultOption }) => {
             />
           </div>
 
-          {/* Pool reward of duration */}
+          {/* Estimated pool rewards */}
           <div className="d-flex align-items-center mt-4 w-100">
+            <div className="d-flex align-items-center">
+              <SecondaryText>Your estimated rewards </SecondaryText>
+            </div>
+            <PoolRewardData className="ml-auto" color={color}>
+              {renderEstimatedRewards()} RBN
```

Comment:
Can you round up the number to nearest integer so we can remove the decimal place entirely?

Reply:
Done!